src/ProjectService.Data.Provider.MsSql.Ef/Migrations/20210419235819_UpdateProjectsAndProjectsUsersTables.cs

```diff
@@ -0,0 +1,81 @@
+﻿using LT.DigitalOffice.ProjectService.Models.Db;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+
+namespace LT.DigitalOffice.ProjectService.Data.Provider.MsSql.Ef.Migrations
+{
+    [DbContext(typeof(ProjectServiceDbContext))]
+    [Migration("20210419235819_InitialCreate")]
```

Comment:
Are you sure what it's correct migration name?
I think it sould be "20210419235819_UpdateProjectsAndProjectsUsersTables"

Reply:
Fixed